erikiado/jp2_online
# Task for the widget of logout

Checks if there is some log out widget in the page.

#### What's this PR do?
Verify if exists some log out widget in the page when you are log in, and verify if doesn't exists for not logged users
#### Where should the reviewer start?
Revieweer should start in tosp_auth/test.py, because the functionality was already in the project. this pull request only checks if exist a Log out in jp2_online/templates/base/home.html.

#### How should this be manually tested?
When you enter to the url you souldn't see a Log out button.
With your super user access to url/admin do a log in and back to the index page.
Verify if exists a Log out button

#### Any background context you want to provide?
Also, verify if I do the correct steps of the process to do code, a commit and a pull request.



This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
